LoopKit/HealthStoreUnitCache.swift

```diff
@@ -29,6 +33,9 @@ public class HealthStoreUnitCache {
         userPreferencesChangeObserver = NotificationCenter.default.addObserver(forName: .HKUserPreferencesDidChange, object: healthStore, queue: nil, using: { [weak self] (_) in
             _ = self?.unitCache.mutate({ (cache) in
                 cache.removeAll()
+                DispatchQueue.global().async {
+                    NotificationCenter.default.post(name: .HealthStoreUnitDidChange, object: healthStore)
```

Comment:
[nit] It might make more sense to have the object be the unit cache if that is possible?